package.json

```diff
@@ -37,4 +38,4 @@
 		"tape": "^4.13.0"
 	},
 	"license": "MIT"
-}
+}
```

Comment:
all files should always have a trailing newline.

Reply:
Fix.

Maybe to always ensure we have this standard it would be nice to have prettier in place